tests/objc-cli/libmanaged/Tests/Tests.m

```diff
@@ -670,6 +670,14 @@ - (void) testHash {
     XCTAssertFalse ([c1 hash] == [c3 hash], "Non-equal objects have different hashes");
 }
 
+- (void) testOperatorOverloading {
+    Methods_OperatorCollision * one = [[Methods_OperatorCollision alloc] initWithVal:1];
+    Methods_OperatorCollision * two = [[Methods_OperatorCollision alloc] initWithVal:2];
+    XCTAssertTrue ([[Methods_OperatorCollision addC1:one c2:two] value] == 3, "1 + 1");
+    XCTAssertTrue ([[Methods_OperatorCollision subtraction:two c2:one] value] == 1, "2 - 1");
```

Comment:
@chamons @spouliot I wonder if we should go:

```
addValue: andValue:
```

```
substractValue: andValue:
```

```
multiplyValue: andValue:
```

Instead of using the argument name.